src/Plugin/Core/AutoloadConnector.php

```diff
@@ -55,7 +56,14 @@ public function linkAutoloader(Event $event)
                 $defaultVendorDir = \Composer\Config::$defaultConfig['vendor-dir'];
 
                 $packagePath = $composer->getInstallationManager()->getInstallPath($package);
-                $jsonFile = new \Composer\Json\JsonFile($packagePath . DIRECTORY_SEPARATOR . 'composer.json', new \Composer\Util\RemoteFilesystem($event->getIO()));
```

Comment:
this shouldn't have been a remote filesystem anyway, so we can simplify that, bu just passing the first arg

Reply:
removed the second argument